shaders/src/light_directional.fs

```diff
@@ -36,14 +36,44 @@ void evaluateDirectionalLight(const MaterialInputs material,
     Light light = getDirectionalLight();
 
     float visibility = 1.0;
+
 #if defined(HAS_SHADOWING)
     if (light.NoL > 0.0) {
+
+        // cast a ray in the direction of the light
```

Comment:
This all should probably go within the `if(frameUniforms.directionalShadows)` block.

Reply:
We will most likely want a different uniform to control contact shadows.  That code should be into shadowing.fs though before the PR is ready for review.

Reply:
yes and yes